### PR TITLE
Update app-configuration-policies-use-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

fixes#https://github.com/MicrosoftDocs/memdocs/issues/3484

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by traya1** [comment]
Learn Build status updates of commit _[219bef3](https://github.com/msbemba/memdocs/commits/219bef365b7f381cea9865ff2904c552ffa76f38)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/intune/apps/app-configuration-policies-use-ios.md](https://github.com/msbemba/memdocs/blob/patch-190/memdocs/intune/apps/app-configuration-policies-use-ios.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202301102059027239-3501/BuildReport?accessString=939a173e00e26b60cd0783c8a0cbc55ff90ecf2b48a8252630f294ae13ec0d8b).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

File: memdocs/intune/apps/app-configuration-policies-use-ios.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ Additionally, Intune supports the following token types in the property list:
 
 Apple's Automated Device Enrollments are not compatible with the app store version of the Company Portal app by default. However, you can configure the Company Portal app to support iOS/iPadOS ADE devices even when users have downloaded the Company Portal from the App Store by using the following steps.
 
-1. In [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), add the Intune Company Portal app if it has not been added yet, by going to **Apps** > **All apps** > **Add** > **Managed devices**.
+1. In [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), add the Intune Company Portal app if it has not been added yet, by going to **Apps** > **All apps** > **Add** > **Ios Store App**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-configuration-policies-use-ios.md:227]:
```suggestion
1. In [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), add the Intune Company Portal app if it has not been added yet, by going to **Apps** > **All apps** > **Add** > **iOS Store App**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**